backend/src/vehicles/entities/vehicle.entity.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  Unique,
+} from 'typeorm';
+import { Organization } from '../../organizations/entities/organization.entity';
+
+@Entity()
+@Unique(['licensePlate'])
+export class Vehicle {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  licensePlate: string;
+
+  @Column()
+  notes?: string;
```

Comment:
Also not your fault. Probably you'll have to update some additional types in other places of the app
```suggestion
  @Column({ nullable: true, type: 'text' })
  notes: string | null;
```